src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -17,6 +17,7 @@ function Checkboxes ($module) {
     return this
   }
 
+  /** @type {NodeListOf<HTMLInputElement>} */
```

Comment:
Thanks to the generic types, we can now set types on the `NodeList` being iterated on early, and know they'll carry through...

Reply:
I love this "typed-query-selector". Very happy to have that instead of the comments 😄 Why would you prefer waiting for the TypeScript issue to be resolved? 

If we want to wait though, I'd prefer the casting comments. Yes, they can have human errors (just like our tests can as well), but:
- them being close to the selector means gives them a chance to be picked on in code review
- it's one of those cases were we're actually certain of the type, so the `instanceof` is unnecessary defensiveness for the sake of TypeScript happiness.

Reply:
Yeah give the extended types a try then, might not even need `@type`

(Abandoned Gulp plugins have ruined me, all I'll say for avoiding extra packages 😆)